go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -251,11 +252,11 @@ func (oc *Controller) addEgressFirewallRules(hashedAddressSetNameIPv4, hashedAdd
 func (oc *Controller) createEgressFirewallRules(priority int, match, action, externalID string) error {
 	logicalSwitches := []string{}
 	if config.Gateway.Mode == config.GatewayModeLocal {
-		nodes, err := oc.kube.GetNodes()
+		nodes, err := oc.watchFactory.GetNodes()
```

Comment:
this should be part of your first commit

Reply:
ACK, I will squash